triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerRunnerConfig.scala

```diff
@@ -34,6 +40,8 @@ object TriggerRunnerConfig {
       maxRetries = 6,
       maxSubmissionRequests = 100,
       maxSubmissionDuration = 5.seconds,
+      inFlightCommandBackPressureCount = 1000,
+      inFlightCommandOverflowCount = 10000,
```

Comment:
From a backwards compatibility PoV: should we have a CLI flag to disable the use of overflows?

Reply:
I think it is a good idea to have a flag, not necessary for backward compatibility.

Reply:
I've added a flag to disable overflow checking and, in addition, I've added the ability to alter the overflow count from the CLI.